AUTHORS.rst

```diff
@@ -1,35 +1,28 @@
 ..
-    This file is part of RERO ILS.
-    Copyright (C) 2017 RERO.
+    RERO ILS
+    Copyright (C) 2019 RERO
 
-    RERO ILS is free software; you can redistribute it
-    and/or modify it under the terms of the GNU General Public License as
-    published by the Free Software Foundation; either version 2 of the
-    License, or (at your option) any later version.
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, version 3 of the License.
 
-    RERO ILS is distributed in the hope that it will be
-    useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-    General Public License for more details.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
 
-    You should have received a copy of the GNU General Public License
-    along with RERO ILS; if not, write to the
-    Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
-    MA 02111-1307, USA.
-
-    In applying this license, RERO does not
-    waive the privileges and immunities granted to it by virtue of its status
-    as an Intergovernmental Organization or submit itself to any jurisdiction.
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 Credits
 =======
 
-  * Johnny Mariéthoz
-  * Igor Milhit
-  * Peter Weber
-  * Bertrand Zuchuat
   * Aly Badr
-  * Gianni Pante
   * Nicolas Labat
+  * Johnny Mariéthoz
+  * Igor Milhit
   * Miguel Moreira
+  * Gianni Pante
   * Nicolas Prongué
+  * Peter Weber
+  * Bertrand Zuchuat
```

Comment:
Add Sébastien?

Reply:
You're right. I'll add @sebastiendeleze in a future commit.